crt_portal/cts_forms/templates/forms/report_details.html

```diff
@@ -16,7 +16,7 @@
       <span id="display_count">500</span>
       <span id="count_message">word(s) remaining</span>
     </em>
-    <div id="word-limit-alert" class="usa-alert usa-alert--warning usa-alert--slim" >
+    <div id="word-limit-alert" class="usa-alert usa-alert--warning usa-alert--slim" hidden>
```

Comment:
By moving the "hidden" functionality from JS to HTML, we make sure that the default UI state makes sense even if JS is turned off or fails to load.